src/modules/auto-toggle-class-on-scroll.js

```diff
@@ -20,9 +20,9 @@
 	var ATTR_PREFIX = 'data-toggle-class-on-scroll-';
 
 	var ATTR_ADD_BEFORE = ATTR_PREFIX + 'add-before';
-	var ATTR_REM_BEFORE = ATTR_PREFIX + 'rem-before';
+	var ATTR_REM_BEFORE = ATTR_PREFIX + 'remove-before';
```

Comment:
Ya right.. will switch back to rem and make sure all other use the same. Will notify team member of the choice done.